app/src/main/java/bg/zahov/app/ui/theme/Theme.kt

```diff
@@ -10,7 +10,10 @@ private val DarkColorScheme = darkColorScheme(
     secondary = less_vibrant_text,
     onPrimary = less_vibrant_text,
     onSecondary = white,
-    background = background
+    background = background,
```

Comment:
You should use the [theme builder](https://m3.material.io/blog/material-theme-builder) instead to create dark and light themes for your app and use them throughout all views.